kex.c

```diff
@@ -965,6 +968,12 @@ kex_choose_conf(struct ssh *ssh)
 		    newkeys->enc.name,
 		    authlen == 0 ? newkeys->mac.name : "<implicit>",
 		    newkeys->comp.name);
+		send_encryption_telemetry(ctos ? "ctos" : "stoc",
```

Comment:
This is common between ssh client and ssh server. Inside the "send-encryption_telemetry" add logic to specify if it's coming from sshd.exe / ssh.exe.

Reply:
Let's add processname sshd.exe / ssh.exe in all the telemetry functions. It will be easy if we want to analyze client, server logs individually.

Reply:
The _appVer field includes the executable, and it's automatically collected with all Windows telemetry events. Let me know if that is sufficient.